python/dgllife/utils/complex_to_graph.py

```diff
@@ -56,6 +58,148 @@ def get_atomic_numbers(mol, indices):
         atomic_numbers.append(atom.GetAtomicNum())
     return atomic_numbers
 
+def int_2_one_hot(a):
+    """Convert integer encodings on a vector to a matrix of one-hot-encoding
+    """
+    n = len(a)
+    b = np.zeros((n, a.max()+1))
+    b[np.arange(n), a] = 1
+    return b
+
+def PN_graph_construction_and_featurization(ligand_mol,
+                                        protein_mol,
+                                        ligand_coordinates,
+                                        protein_coordinates,
+                                        max_num_ligand_atoms=None,
+                                        max_num_protein_atoms=None,
+                                        max_num_neighbors=4,
+                                        distance_bins = [1.5, 2.5, 3.5, 4.5],
+                                        strip_hydrogens=False):
+    """Graph construction and featurization for `PotentialNet for Molecular Property Prediction
+     <https://pubs.acs.org/doi/10.1021/acscentsci.8b00507>`__.
+
+    Parameters
+    ----------
+    ligand_mol : rdkit.Chem.rdchem.Mol
+        RDKit molecule instance.
+    protein_mol : rdkit.Chem.rdchem.Mol
+        RDKit molecule instance.
+    ligand_coordinates : Float Tensor of shape (V1, 3)
+        Atom coordinates in a ligand.
+    protein_coordinates : Float Tensor of shape (V2, 3)
+        Atom coordinates in a protein.
+    max_num_ligand_atoms : int or None
+        Maximum number of atoms in ligands for zero padding, which should be no smaller than
+        ligand_mol.GetNumAtoms() if not None. If None, no zero padding will be performed.
+        Default to None.
+    max_num_protein_atoms : int or None
+        Maximum number of atoms in proteins for zero padding, which should be no smaller than
+        protein_mol.GetNumAtoms() if not None. If None, no zero padding will be performed.
+        Default to None.
+    distance_bins : list of float
```

Comment:
Interchange the order of the doc for distance_bins and the doc for max_num_neighbors for a consistent order.